internal/pkg/schema-registry/client.go

```diff
@@ -47,6 +47,11 @@ func (c *Client) GetTopLevelConfig() (srsdk.Config, error) {
 	return res, err
 }
 
+func (c *Client) DeleteTopLevelConfig() (string, error) {
+	res, _, err := c.DefaultApi.DeleteTopLevelConfig(c.context)
```

Comment:
So these APIs return a JSON string? Can we update the SDK to unmarshal this string into a struct?

Reply:
Their team don't want to make this breaking change of api 😕